cypress/support/pages/settings.pageObjects.js

```diff
@@ -0,0 +1,36 @@
+import PageObject from '../PageObject';
+
+
+class settingsPageObject extends PageObject {
+  url = '/settings/';
+
+  get visit() {
+    return cy.visit('/settings/');
+  }
+  get usernameField(){
+    return cy.getByDataCy('settings-username');
+  }
+
+  get updateBtn() {
```

Comment:
methods with actions should say what they do clickUpdtBtn(), updtUsernameField() etc.